Tasks/LambdaInvokeFunction/TaskOperations.ts

```diff
@@ -49,6 +53,17 @@ export class TaskOperations {
         }
     }
 
+    // A quick heuristic for if it is json. In the past, it was possible to put in just a string
+    // which is not valid JSON. This heurstic checks for if it is a JSON string, object, or array
+    private isJson(input: string): boolean {
+        const i = input.trim()
+
+        return (
+            (i.startsWith('"') || i.startsWith('{') || i.startsWith('[')) &&
+            (i.endsWith('"') || i.endsWith(']') || i.endsWith('}'))
+        )
```

Comment:
Should this match starting and ending punctuation? e.g. would it make sense for this to start with a `"` and end with a `}`? Also, if this is supporting quoted strings, should you include single-quotes?

Reply:
Single quotes are not valid JSON. The reason I didn't check matching pairs is:
```
[{"abc": "bcd"}
```
is invalid JSON but if it gets stringed it will be way more confusing what the issue is

Reply:
Could you just message something like "Payloads starting in '[' must end in ']'"?

Also: are numbers considered valid JSON?

Reply:
Payloads must be JSON, see the updated task form change. Numbers being JSON true, but it would break backwards comparability because they are currently getting wrapped in quotes